lms/djangoapps/ccx/migrations/0002_convert_memberships.py

```diff
@@ -0,0 +1,99 @@
+# -*- coding: utf-8 -*-
+from south.utils import datetime_utils as datetime
+from south.db import db
+from south.v2 import DataMigration
+from django.db import models
+
+class Migration(DataMigration):
+
+    def forwards(self, orm):
+        "Convert CCX Memberships to Course Enrollments."
+        from ccx_keys.locator import CCXLocator
+        for membership in orm['ccx.CcxMembership'].objects.all():
+            #ccx = orm['ccx.CustomCourseForEdx'].objects.get(pk=membership.ccx)
+            ccx = membership.ccx
+            course_key = CCXLocator.from_course_locator(ccx.course_id, ccx.id)
+            enrollment, created = orm['student.CourseEnrollment'].objects.get_or_create(
+                    user=membership.student,
+                    course_id=course_key,
+            )
+            if created:
+                enrollment.save()
```

Comment:
If `created` is `True`, the entry is already there -- you don't have to `save()` again.